integration-tests/tests/api/target/usage.spec.ts

```diff
@@ -2681,11 +2669,7 @@ test.concurrent(
     );
     client.collectUsage()(
       {
-        document: parse(/* GraphQL */ `
-          query {
-            a
-          }
-        `),
+        document: parse('{ a }'),
```

Comment:
Should this still have a `query` in it?

Reply:
I don’t think since all tests pass cc @kamilkisiela @n1ru4l